.github/ISSUE_TEMPLATE/issue.md

```diff
@@ -4,14 +4,14 @@ about: Create a report to help us improve
 
 ---
 
-**Version of the cusom_component**
-<!-- If you are not using the nevest version, download and try that before opening a issue-->
+**Version of the cutsom_component**
```

Comment:
```suggestion
**Version**
```
Copy+paste fail... this is not a custom_component this is a python module... my bad...

I have added a suggestion here, if you accept that change I can merge it :)